src/Language/Fixpoint/Horn/Solve.hs

```diff
@@ -43,11 +48,11 @@ solveHorn cfg = do
            else pure cfg
   cfg <- F.withPragmas cfg opts
 
-  r <- solve cfg q
+  r <- solve @Void cfg q
```

Comment:
This is safe since it is loaded from a file and can never be a `AS`?

Reply:
I'm not sure if instantiating the type to @Void implements the correct behavior. However, it is necessary here since the signature of solve horn:
`solveHorn :: F.Config -> IO ExitCode `
does not include s.
I believe it could be fixed by using s as a phantom type, but I'm not sure if that's something I'm supposed to change.

Reply:
I'm not sure either, but let's leave it for now and see what happens.